spec/virtual_delegates_spec.rb

```diff
@@ -236,14 +236,50 @@ def self.connection
   end
 
   context "with polymorphic has_one" do
+    it "supports select" do
+      Author.delete_all
+      Book.delete_all
```

Comment:
I don't understand the need to delete authors and books in these tests. I don't see other tests creating them.  Do we have some setup that creates these and we're trying to remove them before we create the objects we want to test?